phpcs.xml

```diff
@@ -10,6 +10,9 @@
 	</rule>
 	<rule ref="TribalScents"/>
 
+	<rule ref="PHPCompatibility"/>
```

Comment:
```suggestion
	<rule ref="PHPCompatibilityWP"/>
```

(Otherwise you still won't benefit from the PHPCompatibilityWP ruleset taking polyfilled PHP feature in WP into account)